src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String name, String color) {
+        super(name, color);
+        this.radius = (int)(Math.random() * 100);
```

Comment:
let's pass values through constructor or set with setters

Reply:
what about this? https://github.com/mate-academy/jv-oop-advanced/pull/182/files/eb7f9215c750fc65fabe6669c228ca4890b65d5c..82c637f5462596c5c5ce51fb29f475868ce503ed#r691183181
through constructor, not in constructor